### DISCUSSION TITLE
Form widgets get lost after deleting the .qgs project file (QGIS SUPPORT)

### DISCUSSION THREAD

**Discussion by enheduanna** [opengisch/QField#2629] [Q&A]
HI
I have created a lot of shape files of archaeological sites to be surveyed. I created widgets for the different field in the attribute table but now they are all gone. How is that possible. When I opened the shape file, its as if I had never set it up.
Thank you.

**Comment by suricactus** [comment]
The widget configuration is stored within the `.qgs`/`.qgz` file, not within the `.shp` file. Shapefiles are used only to store geographical and attribute data and nothing related to the visualization.

Also note this is a QField ecosystem forum, not general purpose QGIS support. Check gis.stackexchange.com, QGIS telegram channels or other various general QGIS support channels.

**Reply by enheduanna** [reply]
Thank you - got it and if I delete the qgs file then goes all the widgets that I have set up?

**Reply by suricactus** [reply]
Yes.

**Reply by enheduanna** [reply]
OMG - I have been deleting those project files when reorganizing.

**Reply by suricactus** [reply]
Please find a better QGIS support forum to resolve your issues and let us know if you experience problems after carefully reading the QField/QFieldCloud documentation.
